src/planscape/impacts/filters.py

```diff
@@ -17,15 +17,18 @@ class TreatmentPlanFilterSet(filters.FilterSet):
     scenario = filters.ModelChoiceFilter(
         field_name="scenario",
         queryset=get_scenarios_for_filter,
+        help_text="Scenario ID.",
```

Comment:
Indeed this variable will represent the object `scenario'  during the execution inside Python.
However, on the API documentation it will be represented by the ID of the Scenario only.
![Screenshot from 2024-09-17 13-27-27](https://github.com/user-attachments/assets/d1d0c023-f2c7-4279-bd0d-8462f8279036)